test/test_xsort.cpp

```diff
@@ -128,6 +128,56 @@ namespace xt
         }
     }
 
+    TEST(xsort, argsort_stable)
+    {
+        xt::xarray<double> a = {
```

Comment:
```suggestion
        xt::xarray<int> a = {
```
and removing all the `.`

Reply:
Done in 675a42700850b5e5ad6d47599329b4ccfc9def99 and 37e2592f460aceaab3494ffafc68a1ed0a8deb02